lib/routers/establishment/project-versions.js

```diff
@@ -136,6 +137,7 @@ router.get('/:versionId',
       const est = await req.models.Establishment.query().findById(transferToEstablishment).select('name');
       req.version.data.transferToEstablishmentName = est.name;
     }
+    req.version.raReasons = getRaReasons(req.version.data);
```

Comment:
I'm not sure what that function does exactly, but is there a case for only calling the function if the version requires an RA?

Reply:
Note: I know what it _does_, just not how potentially expensive it is to run on versions that we know don't need RA.

Reply:
sorted.